DNN Platform/Library/Entities/IPFilter/IPFilterController.cs

```diff
@@ -256,5 +261,20 @@ public bool IsAllowableDeny(string currentIP, IPFilterInfo ipFilter)
         }
 
         #endregion
+
+        private static void AssertValidIPFilter(IPFilterInfo ipFilter)
+        {
+            IPAddress parsed;
+            if (IPAddress.TryParse(ipFilter.IPAddress, out parsed) == false)
+            {
+                throw new ArgumentException("IP address is not in correct format");
+            }
+
+            bool isIPRange = string.IsNullOrEmpty(ipFilter.SubnetMask);
+            if (isIPRange && IPAddress.TryParse(ipFilter.SubnetMask, out parsed) == false)
+            {
+                throw new ArgumentException("Subnet mask is not in correct format");
```

Comment:
Ditto

Reply:
Hi @tpluscode , for the localization, you can check DNN Platform\Library\Common\Requires.cs line 102 for example. the localization strings can saved in Website\App_GlobalResources\Exceptions.resx.